rcsb/utils/dictionary/DictionaryApiProviderWrapper.py

```diff
@@ -44,6 +44,10 @@ def __init__(self, cachePath, useCache=True, cfgOb=None, configName=None, **kwar
         self.__configName = configName
         self.__contentInfoConfigName = "content_info_helper_configuration"
         self.__dictLocatorMap = self.__cfgOb.get("DICT_LOCATOR_CONFIG_MAP", sectionName=self.__contentInfoConfigName)
+        # self.__dictLocatorMap: dictionary of configured dictionary name mappings, e.g.:
+        #   ordereddict([('pdbx_comp_model_core', ['PDBX_DICT_LOCATOR', 'RCSB_COMP_MODEL_DICT_LOCATOR', 'MA_DICT_LOCATOR']),
+        #                ('pdbx', ['PDBX_DICT_LOCATOR', 'RCSB_DICT_LOCATOR']),
+        #                ('pdbx_core', ['PDBX_DICT_LOCATOR', 'RCSB_DICT_LOCATOR', 'VRPT_DICT_LOCATOR']), ...
```

Comment:
Is commenting this out intended? Is it related to this fix?

Reply:
Yes that was intentional--I was just adding documentation to provide an example of what that object looked like and contained. Not related to any fix--purely documentation purposes.

Reply:
Ok, thanks for explaining